indexing-hadoop/src/main/java/io/druid/indexer/hadoop/DatasourceIngestionSpec.java

```diff
@@ -108,19 +112,66 @@ public QueryGranularity getGranularity()
     return metrics;
   }
 
+  @JsonProperty
+  public boolean getIgnoreWhenNoSegments()
```

Comment:
@b-slim it wouldn't fail @xvrl just meant to rename it so that it is consistent with other config classes where boolean variable getters are called isXXX() and not getXXX().